test/support/step-definitions.js

```diff
@@ -20,7 +21,7 @@ module.exports = (function () {
      * define step library
      */
     steps.forEach(function (step) {
-        require(path.join(stepsFiles, step)).call(library, dictionary);
```

Comment:
glob.sync (line13) returns absolute file paths rather than the relative paths of fs.readdirSync(), so the join is no longer required.